### PR TITLE
SAK-46725 Resources: Plus sign should be in Other Sites folders with tree view available

### PR DESCRIPTION
**Pull request by josecebe**
https://sakaiproject.atlassian.net/browse/SAK-46725

### CODE REVIEW

**Review by feralvarez95 — APPROVED** [review]
I have tested in my local instance, now looks fine!

image

**Review by jesusmmp — APPROVED** [review]
LGTM!

File: library/src/morpheus-master/sass/modules/tool/resources/_resources.scss [hunk 1]
```diff
@@ -259,37 +259,33 @@
...
+		position: relative;
 		&::after {
 			color: var(--sakai-text-color-inverted);
 			content: "+";
 			display: inline-block;
+			font-family: monospace;
+			font-size: 11px;
 			position: absolute;
-			margin: 0.45em 0 0 -2.2em;
+			right: 4px;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/resources/_resources.scss:271]: Please use $standard-space:

```suggestion
			right: $standard-space;
```

File: library/src/morpheus-master/sass/modules/tool/resources/_resources.scss [hunk 2]
```diff
@@ -259,37 +259,33 @@
...
+		position: relative;
 		&::after {
 			color: var(--sakai-text-color-inverted);
 			content: "-";
 			display: inline-block;
+			font-family: monospace;
+			font-size: 11px;
 			position: absolute;
-			margin: 0.55em 0 0 -2.2em;
+			right: 4px;
```
Comment (fostersdesign) [inline library/src/morpheus-master/sass/modules/tool/resources/_resources.scss:286]: Please use $standard-space:

```suggestion
			right: $standard-space;
```

**Review by fostersdesign — CHANGES_REQUESTED** [review]

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Thanks for the review @fostersdesign and good job @josecebe

**Pull request merged**